projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-util.ts

```diff
@@ -82,7 +82,8 @@ export class PivotUtil {
         pivotKeys: IPivotKeys,
         lvl: number,
```

Comment:
I am also not sure if this check for the combined lvl depth is needed, since we go in depth of the current dimension, not all combined.